jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/DefaultServlet.java

```diff
@@ -395,6 +396,15 @@ else if (gzip == Boolean.TRUE)
         return ret;
     }
 
+    @Override
+    public String getInitParameter(String name)
```

Comment:
Add some javadoc to say that it also looks for the prefix version of the name as a context attribute.